css/index.css

```diff
@@ -48,99 +129,270 @@ table {
 }
 
 * {
-    box-sizing: border-box;
+	box-sizing: border-box;
 }
 
-html, body {
-    height: 100%;
-    font-family: 'Roboto', sans-serif;
+html,
+body {
+	height: 100%;
+	font-family: 'Roboto', sans-serif;
 }
 
-h1, h2, h3, h4, h5 {
-    font-size: 18px;
-    margin-bottom: 15px;
-    font-family: 'Rubik', sans-serif;
+h1,
+h2,
+h3,
+h4,
+h5 {
+	font-size: 18px;
+	margin-bottom: 15px;
+	font-family: 'Rubik', sans-serif;
 }
 
 p {
-    line-height: 1.4;
+	line-height: 1.4;
+}
+
+/* Header for both the Home and About Pages */
+header {
+	display: flex;
+	flex-direction: row;
+	width: 100%;
+	padding: 0 3%;
+	padding-top: 1%;
+}
+
+header img {
+	width: 15%;
+	margin: 1%;
+}
+
+header nav {
+	display: flex;
+	flex-direction: row;
+	justify-content: space-evenly;
+	align-items: center;
+	width: 85%;
+	margin: 1%;
 }
 
+header nav a {
+	text-decoration: none;
+	color: black;
+	padding: 1%;
+}
+
+header nav a:hover {
+	background-color: black;
+	color: white;
+}
+
+.streetImage img {
+	width: 100%;
+	padding: 3%;
+}
+
+/* CSS for the Home Page Starts Here */
 .container {
-    width: 800px;
-    margin: 0 auto;
+	width: 800px;
+	margin: 0 auto;
 }
 
 .top-content {
-    display: flex;
-    flex-wrap: wrap;
-    justify-content: space-evenly;
-    margin-bottom: 20px;
-    border-bottom: 1px dashed black;
+	display: flex;
+	flex-wrap: wrap;
+	justify-content: space-evenly;
+	margin-bottom: 20px;
+	border-bottom: 1px dashed black;
 }
 
 .top-content .text-container {
-    width: 48%;
-    padding: 0 1%;
-    padding-bottom: 20px;  
+	width: 48%;
+	padding: 0 1%;
+	padding-bottom: 20px;
 }
 
 .middle-content {
-    margin-bottom: 20px;
-    border-bottom: 1px dashed black;
+	margin-bottom: 20px;
+	border-bottom: 1px dashed black;
 }
 
 .middle-content h2 {
-    padding: 0 2%;
-    margin-bottom: 0;
+	padding: 0 2%;
+	margin-bottom: 0;
 }
 
 .middle-content .boxes {
-    display: flex;
-    flex-wrap: wrap;
-    justify-content: space-evenly;
+	display: flex;
+	flex-wrap: wrap;
+	justify-content: space-evenly;
+	padding: 1%;
+}
+
```

Comment:
The parent .box also sets the padding and margin, so the child will inherit those styles so they can be omitted in each individual box.